modules/ROOT/pages/index.adoc

```diff
@@ -1,39 +1,205 @@
+components.
+* Within a Mule app, event processing components are arranged into one or
+more container-like components, called <<flows>>.
++
+For more on this topic, see xref:about-mule-event.adoc[Mule Events].
 
-* Deploy or integrate apps or systems on premise or in the Cloud.
-* Employ out-of-the-box connectors to create SaaS integration apps.
-* Build and expose APIs.
-* Consume APIs.
-* Create Web services that orchestrate calls to other services.
-* Create interfaces to expose apps for mobile consumption.
-* Integrate B2B with solutions that are secure, efficient, and quick to build and deploy.
-* Shift apps to the Cloud.
-* Connect B2B e-commerce activities.
+[[flows]]
+== Flows
 
-== About the Non-blocking Execution Engine
+Understanding basic flow architecture is key to understanding Mule.
+Essentially, every Mule flow contains a series of components that can
+receive or process messages.
 
-The Mule execution engine is based on a non-blocking runtime. This is a task-oriented execution model that allows you to take advantage of non-blocking IO calls and to avoid performance problems.
+.Simple Flow in Design Center
+image::flow-example.png[Simple Flow in Design Center]
 
-Flows always function synchronously. If you need to achieve asynchronous patterns, such as Fire-and-Forget, you can use the Async (`<async>`) component.
+At the simplest level, flows are sequences of Event processors. A message
+that enters a flow can pass through a variety of processors. In a typical
+flow, a Mule app receives a message through an Event source (an endpoint, such
+as an HTTP listener), transforms that message into a new format, and processes
+any business logic before writing the processed message to an external system
+in a format that the system can read.
 
-Each Mule event processor can inform the runtime if it is CPU intensive, CPU light, or an IO intensive operation. This helps the runtime self-tune for different workloads dynamically, removing the need for you to manage thread pools manually to achieve optimum performance.
+//TODO: VERIFY INFO ON WAYS FLOW CAN CALL OTHER FLOWS
+To break up processing into more manageable units, Mule apps often contain
+multiple, interrelated flows, instead of just a single flow. A flow can call
+another flow as a direct reference, or it can call a flow through a common communication protocol or method, such as JMS, HTTP, FTP, or File. The called flows might be part of the same Mule app or might run in a separate deployment on another Mule runtime across the network.
 
-== About Thread Pool Executor (Schedulers) Configuration
+For more on this topic, see xref:about-flows[Flows and Subflows] and
+xref:mule-application-about.adoc[Mule Applications].
 
-You can configure schedulers (thread pool executors) through the well-commented `schedulers.conf` file in the standalone Mule EE installation.
+== Event Sources
 
-== See Also
+An Event source (or Trigger) is the first processor in a flow. This 
```

Comment:
`processor` -> `element`. Sources generate events, they don't process them.

Reply:
changing to "component"

Reply:
done, but used "component"